ch2driver/pytpcc/constants.py

```diff
@@ -172,9 +172,9 @@
     "HAND_OPTIMIZED_QUERIES":0,
     "NON_OPTIMIZED_QUERIES":1
 }
-CH2PP_CUSTOMER_EXTRA_FIELDS = 128
-CH2PP_ORDERS_EXTRA_FIELDS = 128
-CH2PP_ITEM_EXTRA_FIELDS = 128
+CH2_CUSTOMER_EXTRA_FIELDS = 128
+CH2_ORDERS_EXTRA_FIELDS = 128
+CH2_ITEM_EXTRA_FIELDS = 128
```

Comment:
Maybe instead of having a single default value for each of the collections, we can relate it to the schema.

Something like this perhaps (at least logically - there is probably a better implementation):
```
CUSTOMER_EXTRA_FIELDS = {
    CH2_DRIVER_SCHEMA["CH2"] = 0,
    CH2_DRIVER_SCHEMA["CH2P"] = 0,
    CH2_DRIVER_SCHEMA["CH2PP"] = 128,
}
```
and likewise for `orders` and `item` collections.

Then this could be used like so elsewhere (e.g. in `NestcollectionsDriver`):
```
constants.CUSTOMER_EXTRA_FIELDS[self.schema]
```